app/src/main/java/com/example/playlistmaker/common/di/PresentationModule.kt

```diff
@@ -10,10 +13,22 @@ import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.module
 
 val viewModelModule = module {
+
     viewModel { SettingsViewModel(get(), get(), get()) }
     viewModel { SearchViewModel(get(), get(), get()) }
     viewModel { PlayerViewModel(get(), get(), get(), get()) }
     viewModel { PlaylistsViewModel(get()) }
     viewModel { FavoritesViewModel(get(), get()) }
-    viewModel { CreatePlaylistViewmodel(get(), get()) }
+
+    viewModel { (playlistId: String) ->
+        CreatePlaylistViewmodel(playlistId = playlistId, get(), get())
+    }
+
+    viewModel { (playlistId: String) ->
+        PlaylistViewModel(playlistId = playlistId, get(), get(), get())
+    }
+}
+
+val uimodule = module {
+    factory { (context: Context) -> ConfirmationPresenter(context = context) }
```

Comment:
🍏 Рекомендую переименовать ConfirmationPresenter, так как создается впечатление, что здесь применяется паттерн MVP.
Например:
```suggestion
    factory { (context: Context) -> ConfirmationDialog(context = context) }
```